src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -199,19 +200,23 @@ private void InterceptCallAndAdjustArguments(
             // When the original call is to an instance method, and the interceptor is an extension method,
             // we need to take special care to intercept with the extension method as though it is being called in reduced form.
             Debug.Assert(receiverOpt is not BoundTypeExpression || method.IsStatic);
-            var needToReduce = receiverOpt is not (null or BoundTypeExpression) && interceptor.IsExtensionMethod; // Tracked by https://github.com/dotnet/roslyn/issues/76130: Test this code path with new extensions
-            var symbolForCompare = needToReduce ? ReducedExtensionMethodSymbol.Create(interceptor, receiverOpt!.Type, _compilation, out _) : interceptor; // Tracked by https://github.com/dotnet/roslyn/issues/76130 : test interceptors
+            bool receiverIsValue = receiverOpt is not (null or BoundTypeExpression);
+            bool needToReduceInterceptor = receiverIsValue && interceptor.IsExtensionMethod;
 
-            if (!MemberSignatureComparer.InterceptorsComparer.Equals(method, symbolForCompare))
+            MethodSymbol? interceptorForCompare =
+                interceptor.GetIsNewExtensionMember() && invokedAsExtensionMethod ? interceptor.TryGetCorrespondingExtensionImplementationMethod() :
```

Comment:
>interceptor.TryGetCorrespondingExtensionImplementationMethod() [](http://example.com/codeflow?start=84&length=62)

This looks suspicious. In general getting implementation method is moving us to opposite direction by comparison to what ```ReducedExtensionMethodSymbol.Create``` does on the other branch.